css.md

```diff
@@ -120,7 +120,21 @@ Set a root `font-size` of `62.5%` on the `html` selector, which is roughly equiv
 ## Stacking Order
 
 ### Z-index
-Use variables or maps to manage z-index values. Variables or keys should be named semantically, such as "z-index-modal". Values should ideally be set `1, 2, 3, 4` etc. and shouldn't be set arbitrarily. Declarations like `z-index: 9999;` should never be used and always avoid `!important`. Store all variables or maps in a partial or single place.
+Use variables or maps to manage z-index values. Variables or keys should be named semantically, such as "z-index-modal". Values should ideally be set `1, 2, 3, 4` etc. and shouldn't be set arbitrarily. Store all variables or maps in a partial or single place.
+
+These z-index variables should be used as semantically named. For example, `.c-search__filters` should not be using the `nav` z-index variable. Better to have multiple z-index variables with the same value than mix up their usage (causing the kind of confusion the vars were meant to prevent). For example:
+
+```css
+$z-index: (
+  float: 1,
+  ui: 2,
+  nav: 2,
```

Comment:
I always saw these as landmark z-index levels, i.e. "Filters should be on the same level as the nav, therefore `zindex(nav)`" - perhaps the names should be more generalized? nav/indexFilters condensed into `ui` - or is that too generic?

Reply:
The problem comes when bugs are discovered and we're trying to untangle these. It's a bit like using `$border-color` for a background because it's the color you want to use. Sort of. It gets hard to understand if the previous developer meant it as a landmark or simply grabbed the first existing one that made sense.

One way would be to have a variable at the top, like `$z-nav`, then in the map do:

```css
$z-index: (
  float: 1,
  ui: 2,
  nav: $z-nav,
  indexFilters: $z-nav,
  modal: 3
);
```